docs/src/cli/usage.md

```diff
@@ -72,10 +73,10 @@ Results:
 0 parse errors
 ```
 
-**--num-threads** *num-threads*
+**--num-threads** _num-threads_
 
 Specifies the number of threads for selene to use. Defaults to however many cores your CPU has. If you type `selene --help`, you can see this number because it will show as the default for you.
 
-**--pattern** *pattern*
+**--patterns** _patterns_
 
-A [glob](https://en.wikipedia.org/wiki/Glob_(programming)) to match what files selene should check for. For example, if you only wanted to check files that end with `.spec.lua`, you would input `--pattern **/*.spec.lua`. Defaults to `**/*.lua`, meaning "any lua file".
+Comma-separated [globs](<https://en.wikipedia.org/wiki/Glob_(programming)>) to match what files selene should check for. For example, if you only wanted to check files that end with `.spec.lua`, you would input `--patterns **/*.spec.lua`. Defaults to `**/*.lua,**/*.luau`, meaning "any lua/luau file".
```

Comment:
Why the `<>`?

Reply:
This and the whitespace changes were done automatically by my editor